### ISSUE TITLE
City survey infographic / data visualization - partner with City of Houston & Rice University

### ISSUE THREAD

**Issue by MeliRagsDarr** [sketch-city/project-ideas#74]
We partnered with data nerd Bob Stein from Rice University to survey thousands of Houstonians and got some good feedback. I'm hoping there's someone in our SketchCity community that can help me put some cool info graphics (or teach me how to Tableau? 👍). My goal is to give the data we collected back to the citizens in a meaningful deliverable. Please reach out to me if you would like to meet up or help on this! m.ragsdaledarragh at houston tx dot gov. 8323930808

**Comment by fileunderjeff** [comment]
Hi @MeliRagsDarr, that sounds awesome! Thanks for posting here. I would love to collaborate with you on this. Is the dataset something you can share? I also think @evanoneil, @neerajt, @srigdon, @ardouglass also might be interested.

**Comment by evanoneil** [comment]
Definitely interested!

**Comment by MeliRagsDarr** [comment]
Yes! there is data. I need to double check and make sure that I can share it with you guys (don't laugh 😂 I didn't know if I should ask if there's any interest first, or ask if I should share at first it was like chicken or the egg!) but now that I know people are interested and willing now I can go and ask "hey there's this group out there willing to help us can we share the data..."

Melissa Ragsdale Darragh

713-391-6305  

Let's connect! Facebook | Twitter | LinkedIn | Google+

**Comment by fileunderjeff** [comment]
@MeliRagsDarr fantastic! Yes, if we need to keep the data under wraps with a small team, we can do that.

**Comment by antonywu** [comment]
In terms of project scope. What is the project goal of info graphics? Are we trying to extract data like
number of people per square miles? or % of commercial vs % of residential? or number of crimes per square miles? or average HCAD appraisal values per ward/per sub division?

**Comment by paragmchaudhari** [comment]
Is this thread abandoned?
